src/piadm.sh

```diff
@@ -168,6 +179,14 @@ avail() {
 	# Don't print ones old enough to NOT contain piadm(1M) itself.
 	"${CURL[@]}" "${URL_PREFIX}/?limit=1000" | json -ga -c \
 		"this.name.match(/Z$/) && this.name>=\"$activestamp\"" name
+
+	#
+	# Additionally, check to see if we have a Triton CN, where we can
+	# create an ipxe boot. Restrict to searching only locally on-disk.
+	#
+	if [[ -f "$TRITON_IPXE_LKRN" && -f "$TRITON_IPXE_ARCHIVE" ]]; then
```

Comment:
I think we should also include checking bootparams to ensure that neither `headnode=true` nor `smartos=true` are set. I.e., we don't want you to be able to accidentally set your headnode to boot ipxe, because it may never come back.

Reply:
And thinking about this again, we'll definitely want `piadm update ipxe` to move new bits into place from the (presumably) newer local copy.

But, I'm thinking `piadm bootable -i <image> <pool>`, on a Triton node uses the ipxe that it already has, and the specified image will use only the PI.

Reply:
NOTE:  `bootable -i` is now disallowed on Triton CNs.  For the backup PI you get either what you're running, or the "default" PI.